paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/AddPaymentMethodRequirement.kt

```diff
@@ -88,10 +88,8 @@ internal enum class AddPaymentMethodRequirement {
 
 private val PaymentMethodMetadata.supportsMobileInstantDebitsFlow: Boolean
     get() {
-        val paymentMethodTypes = stripeIntent.paymentMethodTypes
-        val noUsBankAccount = USBankAccount.code !in paymentMethodTypes
-        val supportsBankAccounts = "bank_account" in stripeIntent.linkFundingSources
```

Comment:
These conditions are checked in the backend and manifest in `linkSupportedPaymentMethodsOnboardingEnabled`.